preview-start.js

```diff
@@ -1,6 +1,6 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
The `guides` and `guides-index` are not in this repo, so perhaps we should omit them here. I am not too familiar with the preview work, however.

Reply:
I'm hazy on this as well. We'll probably need to use our own templates, but there is still work to be done on the docs platform side as well